packages/pds/tests/invites-admin.test.ts

```diff
@@ -2,7 +2,7 @@ import { TestNetworkNoAppView, SeedClient } from '@atproto/dev-env'
 import AtpAgent from '@atproto/api'
 import { randomStr } from '@atproto/crypto'
 
-describe('pds admin invite views', () => {
+describe.skip('pds admin invite views', () => {
```

Comment:
Maybe we can un-skip now?  If not, might just be worth a note.

Reply:
yup we can 👍 forgot why this was skipped originally